src/celeritas/io/ImportOpticalMaterial.hh

```diff
@@ -92,7 +92,8 @@ struct ImportScintData
     //! Whether all data are assigned and valid
     explicit operator bool() const
     {
-        return static_cast<bool>(material) && resolution_scale >= 0;
+        return (static_cast<bool>(material) || !particles.empty())
```

Comment:
The import bit I was on the fence on that. I left it as an `or` in case someone decides to store everything in their geometry description and decide to use either material or particle sampling at run time.

Reply:
Good point, I'm fine with that. (We'll have to remember to add such a switch or to validate that both aren't there.)